site/content/en/docs/metadata-artifacts.md

```diff
@@ -88,15 +88,15 @@ Decorated PR (finished): "revision":"5dd9241d43f256984358354d1fec468f274f9ac4"
 }
 ```
 
-[`Started.json` PodUtil](https://github.com/kubernetes/test-infra/blob/016edc15b8271c7528993cea0615cb11ecff201c/prow/initupload/run.go#L37)
+[`Started.json` PodUtil](https://github.com/kubernetes-sigs/prow/blob/7013691e3f35afd02f300c04ccd06ebed66a785f/prow/pod-utils/downwardapi/jobspec.go#L213)
```

Comment:
For reviewers' information:
the link points to `func refsToStarted()`,
and the function was moved from `prow/initupload/run.go` to `prow/pod-utils/downwardapi/jobspec.go` in https://github.com/kubernetes/test-infra/pull/27259.